src/app/footer/Footer.js

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import {
+  FaFacebookF,
+  FaInstagram,
+  FaTwitter,
+  FaYoutube,
+  FaLinkedinIn,
+} from "react-icons/fa";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCcVisa, faCcMastercard } from "@fortawesome/free-brands-svg-icons";
+import Image from "next/image";
+import logo from "@/app/icons/logo-small.6645556.svg";
+
+const COMPANY_CATEGORIES = [
```

Comment:
why do need this constant? it does not look like company categories. Bellow you used it, but if you want to used - you don't need this array. you can directly define two constants. Array used for iterations